tests/integration/it_utils.py

```
@@ -633,6 +712,146 @@ def create_mpt_token_and_authorize_source(
+            trading_fee=12,
+        ),
+        lp_wallet,
+        client,
+    )
+    if (
+        not response.is_successful()
+        or response.result.get("engine_result") != "tesSUCCESS"
+    ):
+        raise ValueError(
+            f"AMMCreate transaction failed: {response.result.get('engine_result')}"
+        )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)